Userland/Libraries/LibCore/System.cpp

```diff
@@ -116,6 +116,15 @@ ErrorOr<void> mount(int source_fd, StringView target, StringView fs_type, int fl
     HANDLE_SYSCALL_RETURN_VALUE("mount", rc, {});
 }
 
+ErrorOr<void> umount(StringView mount_point)
+{
+    if (mount_point.is_null())
+        return Error::from_errno(EFAULT);
+
+    int rc = syscall(SC_umount, mount_point.characters_without_null_termination(), mount_point.length());
+    HANDLE_SYSCALL_RETURN_VALUE("umount", rc, {});
+}
+
```

Comment:
This should probably go in a separate commit, I'd say.

Reply:
@IdanHo what do you think? I will split it if needed :)

Reply:
Sure, adding the LibCore wrapper is a separate atomic change that can stand on its own

Reply:
Done :)